src/test/scala/fr/acinq/bitcoin/scalacompat/reference/TransactionSpec.scala

```diff
@@ -65,12 +67,12 @@ class TransactionSpec extends FlatSpec with Matchers {
   import TransactionSpec._
 
   "Bitcoins library" should "pass reference tx valid tests" in {
-    val stream = classOf[ScriptSpec].getResourceAsStream("/data/tx_valid.json")
+    val stream = classOf[scalacompat.ScriptSpec].getResourceAsStream("/data/tx_valid.json")
```

Comment:
Shouldn't this be `TransactionSpec` (and same below)?

Reply:
Yes it would be cleaner, done in https://github.com/ACINQ/bitcoin-lib/pull/63/commits/c67bb611f713af65c71ebea6354db1b9ca951a91